app/src/main/java/com/alphawallet/app/entity/tokens/ERC721Ticket.java

```diff
@@ -411,7 +412,7 @@ public boolean hasArrayBalance()
     }
 
     @Override
-    public List<BigInteger> getArrayBalance() { return balanceArray; }
```

Comment:
@JamesSmartCell under what circumstance would you want the balance with zeros in the array for 721 tickets?

Reply:
This is for erc875. Most likely it's an artifact from the ticket class.

Reply:
But it should be in a different PR as it's unrelated to this one